gg-pac-team3-back/src/models/CourseModel.mjs

```diff
@@ -0,0 +1,83 @@
+import admin from 'firebase-admin';
+
+class CourseModel {
+  constructor({
+    courseId,
+    courseName,
+    courseType,
+    description,
+    imageUrl,
+    logoUrl,
+    duration,
+    rating,
+    otherInfo,
+    videoURL = [], // Default to empty array if not provided                
```

Comment:
Sorry i just noticed this, but `videoURL` should be `videoUrl` for consistency (see `imageUrl`, `logoUrl`)

Reply:
Also, I thought we were going to get rid of this in the CourseModel?